test/test_meshParitionFromTetgenFiles.py

```diff
@@ -1,98 +0,0 @@
-#!/usr/bin/env python
```

Comment:
This is going away because of the spelling error in the file name.  I'll update make check  to point to the correct one.